addons/stock/views/stock_inventory_views.xml

```diff
@@ -212,6 +225,7 @@
         <field name="view_mode">tree,kanban,form</field>
         <field name="view_id" ref="view_inventory_tree"/>
         <field name="search_view_id" ref="view_inventory_filter"/>
+        <field name="domain">['|', ('picking_id', '=', False), ('state', '=', 'done')]</field>
```

Comment:
why?

Reply:
Main reason was to avoid confusion in inventories and prevent manual manipulation on them while they are not done, as the values could on them could seem incoherent if the picking that triggered the inventory is not validated yet. I can add a boolean field on the inventories to avoid using a o2m on the picking though

Reply:
allez ok